### PR TITLE
RHDEVDOCS 6324 Add the need to change pipelines with ClusterTasks to Pipelines 1.17 RNs

### PR DESCRIPTION
**Pull request by mramendi**
Version(s):

pipelines-docs-1.17

Issue:

RHDEVDOCS 6324

Link to docs preview: https://86270--ocpdocs-pr.netlify.app/openshift-pipelines/latest/release_notes/op-release-notes-1-17.html

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

File: modules/op-release-notes-1-17.adoc [hunk 1]
```diff
@@ -158,7 +158,12 @@ spec:
 [id="breaking-changes-1-17_{context}"]
 == Breaking changes
 
-* With this update, the deprecated `ClusterTask` resource is removed from the Operator. As an alternative, you can use the cluster resolver.
+* With this update, the deprecated `ClusterTask` resource is removed from the Operator. As an alternative, you can use the cluster resolver to access tasks thta {pipelines-shortname} installs in the `openshift-pipelines` namespace.
++
+[IMPORTANT]
+====
+Before upgrading to {pipelines-shortname} 1.17, if you configured any pipelines that use `ClusterTask` resources, you must edit them to use the tasks in the `openshift-pipelines` namespace. Otherwise, after the upgrade, the pipelines will fail.
```
[the remark this answers is not in the document]
Comment (mramendi) [reply modules/op-release-notes-1-17.adoc:165]: We don't normally link from RNs to docs but this is an exceptional situation so I will do this.

**Review by vdemeester — APPROVED** [review]

**Comment by savitaashture** [comment]
LGTM

**Comment by kalexand-rh** [comment]
@mramendi, since this sounds like an important clarification, you can get this merged without QE ack. However, you'll still need to get QE to ack this change and open a follow-up PR if they provide additional information.

**Review by srivickynesh — APPROVED** [review]

**Review by lahinson — COMMENTED** [review]
/lgtm

**Pull request merged**